model/reward/instructor/trainer.py

```diff
@@ -178,11 +183,11 @@ def prediction_step(
         args=args,
         loss_function=training_conf["loss"],
         train_dataset=train,
-        eval_dataset=evals,
+        eval_dataset=torch.utils.data.ConcatDataset(evals.values()),
```

Comment:
do we actually want this?

Reply:
A `KeyError` is raised if not, since `evals` is a `dict`, not a `Dataset`. I'm not sure why it was created like this in [`get_datasets`](https://github.com/LAION-AI/Open-Assistant/blob/4250425c94b550df8b9b90082d52923525cbcb7e/model/reward/instructor/utils.py#L121) - I could amend it there?

Reply:
Aha I see, it is created as a dict to follow the sft training I guess, but perhaps it needs some special handling to compute the losses. Maybe we can change this in the future.